tests/testthat/test-unnest.R

```diff
@@ -82,14 +83,15 @@ test_that(".id creates vector of names for grouped data frame unnest", {
     a = data_frame(y = 1),
     b = data_frame(y = 1:2)
   )) %>%
-    dplyr::group_by(x)
+  dplyr::group_by(x)
   out <- unnest(df, .id = "name")
 
   expect_equal(out$name, c("a", "b", "b"))
 })
 
 test_that("can use non-syntactic names", {
-  out <- tibble("foo bar" = list(1:2, 3)) %>% unnest()
+  out <- tibble("foo bar" = list(1:2, 3)) %>%
```

Comment:
single line pipes not yet supported but issue is open ([#74](https://github.com/krlmlr/styler/issues/74))